helpdesk_mgmt_partner_sequence/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    def _prepare_ticket_number(self, values):
+        seq = False
+        if "partner_id" in values or self.partner_id:
```

Comment:
`"partner_id" in values` best with get -> values.get("partner_id")
the function 'get' is faster than 'in'